src/components/CourseInfo/CourseInfo.jsx

```diff
@@ -0,0 +1,40 @@
+import { Link, useParams } from 'react-router-dom';
+
+import Info from '../../common/Info/Info';
+import { getCourseInfo } from '../../helpers/getCourseInfo';
+
+import { mockedCoursesList } from '../../constants';
+import './CourseInfo.css';
+
+export default function CourseInfo() {
+	const { courseId } = useParams();
+	const course = mockedCoursesList.filter(
+		(course) => course.id === courseId
+	)[0];
+	return (
+		<div className='main-container'>
+			<Link className='link' to='/courses'>
+				&larr; Back to courses
+			</Link>
+			<h2>{course.title}</h2>
+			<div className='courseinfo'>
+				<span className='description'>{course.description}</span>
+				<div className='information'>
+					{getCourseInfo(course, ['id', 'duration', 'created']).map(
+						([title, content]) => (
+							<Info key={title} infoTitle={title} infoContent={content} />
+						)
+					)}
+					{getCourseInfo(course, ['authors']).map(([title, content]) => (
```

Comment:
no need to repeat it, once you save necessary params in variable you can just do something like this `<Info
							key={title}
							infoTitle={title}
							infoContent={content}
							multiline={courseParams.find((item) => item === 'authors')}
						/>`
						Thus you will have less code

Reply:
you can simplify it with just `{getCourseInfo(course, [
						'id',
						'duration',
						'creationDate',
						'authors',
					]).map(([title, content]) => (
						<Info
							key={title}
							infoTitle={title}
							infoContent={content}
							multiline={['authors']}
						/>
					))}`

Reply:
I mean no need to use twise getCourseInfo

Reply:
Sorry, at the end of the day I reverted more changes than I should have. Now it's the way it should be :)